.github/workflows/build_wheel.yml

```diff
@@ -0,0 +1,281 @@
+name: Build Wheels
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*"
+  pull_request:
```

Comment:
I don't think this needs to trigger for every pull request or push to main. First, it should only trigger if the `quaddtype` folder gets touched.

But also I think it should only trigger wheel builds if there's a tag like [wheel build] in the commit message or a git tag with a compatible name gets pushed.

Reply:
Also probably worth adding `workflow_dispatch`.